packages/nodes-base/nodes/Webhook.node.ts

```diff
@@ -168,6 +168,11 @@ export class Webhook implements INodeType {
 						value: 'lastNode',
 						description: 'Returns data of the last executed node',
 					},
+					{
+						name: 'No Body Response',
+						value: 'noBodyResponse',
+						description: 'Returns data without a body',
```

Comment:
Could you please make two changes:

- set the name to be `Empty body response`
- set the description to be `Responds with empty body`. I think it would be more clear.

Lastly, move this to be the first option so they are in alphabetical order ;)